client/src/App.jsx

```diff
@@ -1,19 +1,19 @@
-import { useState, useContext } from 'react';
-import reactLogo from './assets/react.svg';
-import './App.scss';
-import Header from './Components/Header/Header';
-import { BrowserRouter, Routes, Route, Navigate } from 'react-router-dom';
-import 'vite/modulepreload-polyfill';
+import { useState, useContext } from "react";
+import "./App.scss";
+import Header from "./Components/Header/Header";
+import { BrowserRouter, Routes, Route, Navigate } from "react-router-dom";
+import "vite/modulepreload-polyfill";
 
-import UserProvider from './Context/UserContext';
-import Discover from './Pages/Discover/Discover';
-import Charts from './Pages/Charts/Charts';
-import About from './Pages/About/About';
-import Landing from './Pages/Landing/Landing';
-import Profile from './Pages/Profile/Profile';
-import Mood from './Pages/Mood/Mood';
+import UserProvider from "./Context/UserContext";
+import Discover from "./pages/Discover/Discover";
```

Comment:
we might need to rename our local repos to lower case 'pages' for these changes. Would need to use `git mv Pages/ pages/